src/main/resources/init_db.sql

```diff
@@ -2,24 +2,76 @@ CREATE SCHEMA `taxi_service` DEFAULT CHARACTER SET utf8;
 
 CREATE TABLE `taxi_service`.`manufacturer`
 (
-    `idManufacturer`      BIGINT(11)   NOT NULL AUTO_INCREMENT,
-    `nameManufacturer`    VARCHAR(225) NOT NULL,
-    `countryManufacturer` VARCHAR(225) NOT NULL,
-    PRIMARY KEY (`idManufacturer`),
-    UNIQUE INDEX `nameManufacturer_UNIQUE` (`nameManufacturer` ASC) VISIBLE
+    `id`      BIGINT(11)   NOT NULL AUTO_INCREMENT,
+    `name`    VARCHAR(225) NOT NULL,
+    `country` VARCHAR(225) NOT NULL,
+    PRIMARY KEY (`id`)
 );
 
-INSERT INTO `taxi_service`.`manufacturer` (`nameManufacturer`, `countryManufacturer`)
+INSERT INTO `taxi_service`.`manufacturer` (`name`, `country`)
 VALUES ('ZAZ', 'UA');
 
-INSERT INTO `taxi_service`.`manufacturer` (`nameManufacturer`, `countryManufacturer`)
+INSERT INTO `taxi_service`.`manufacturer` (`name`, `country`)
 VALUES ('Skoda', 'PL');
 
 ALTER TABLE `taxi_service`.`manufacturer`
-    ADD COLUMN `deleteManufacturer` TINYINT NOT NULL AFTER `countryManufacturer`;
+    ADD COLUMN `deleted` TINYINT NOT NULL AFTER `country`;
+
+CREATE TABLE `taxi_service`.`drivers` (
+                                          `id` BIGINT NOT NULL,
+                                          `name_driver` VARCHAR(225) NOT NULL,
+                                          `licence_number` VARCHAR(225) NOT NULL,
+                                          `deleted` TINYINT NOT NULL DEFAULT 0,
+                                          PRIMARY KEY (`id`),
+                                          UNIQUE INDEX `name_UNIQUE` (`name` ASC) VISIBLE,
+                                          UNIQUE INDEX `licence_number_UNIQUE` (`licence_number` ASC) VISIBLE)
+    ENGINE = InnoDB
+DEFAULT CHARACTER SET = utf8
+COLLATE = utf8_bin;
+
+ALTER TABLE `taxi_service`.`drivers`
+    CHANGE COLUMN `id` `id` BIGINT NOT NULL AUTO_INCREMENT ;
```

Comment:
let's put in this file the `CREATE TABLE` statements that are correct from the start